builtin/credential/okta/path_users.go

```diff
@@ -0,0 +1,139 @@
+package okta
```

Comment:
I can add a similar feature, but I personally think it's preferable to always require a user to be associated with a group and to never allow a user to directly be associated with a policy.